File: be/src/main/java/com/example/Campung/User/dto/LoginResponse.java
package com.example.campung.user.dto;

public class LoginResponse {
    private boolean success;
    private String message;
    private String accessToken;
    
    public LoginResponse(boolean success, String message) {
        this.success = success;
        this.message = message;
    }
    
    public LoginResponse(boolean success, String message, String accessToken) {
        this.success = success;
        this.message = message;
        this.accessToken = accessToken;
    }
    
    public boolean isSuccess() {
        return success;
    }
    
    public void setSuccess(boolean success) {
        this.success = success;
    }
    
    public String getMessage() {
        return message;
    }
    
    public void setMessage(String message) {
        this.message = message;
    }
    
    public String getAccessToken() {
        return accessToken;
    }
    
    public void setAccessToken(String accessToken) {
        this.accessToken = accessToken;
    }
}

File: android/campung/app/src/main/java/com/shinhan/campung/data/remote/response/LoginResponse.kt
package com.shinhan.campung.data.remote.response

import com.shinhan.campung.data.remote.dto.UserDto

//data class LoginResponse(
//    val token: String,
//    val user: UserDto
//)

data class LoginResponse(
    val success: Boolean,
    val message: String,
    val accessToken: String,
    val nickname: String
)
